pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -19,11 +19,31 @@ package v1alpha1
 import (
 	"context"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
 	"knative.dev/pkg/apis"
 )
 
 // Validate EventListener.
 func (t *EventListener) Validate(ctx context.Context) *apis.FieldError {
+	clientset := ctx.Value("clientSet").(dynamic.Interface)
+	if clientset == nil {
+		return apis.ErrMissingField("dynamic clientset")
+	}
+
+	triggerBindings := SchemeGroupVersion.WithResource("triggerbindings")
+	triggerTemplates := SchemeGroupVersion.WithResource("triggertemplates")
+	for _, trigger := range t.Spec.Triggers {
+		_, err := clientset.Resource(triggerBindings).Namespace(t.Namespace).Get(trigger.TriggerBinding.Name, metav1.GetOptions{})
+		if err != nil {
+			return apis.ErrInvalidValue(err, "spec.triggers.TriggerBinding")
```

Comment:
Is there a way to return the actual error back to the user? Or at log it to see what went wrong?

Reply:
Yes, use will get the error when use `kubectl apply` like this:
```
Error from server (BadRequest): error when creating "examples/eventlisteners/eventlistener.yaml": admission webhook "triggers-webhook.tekton.dev" denied the request: mutation failed: invalid value: triggerbindings.tekton.dev "pipeline-binding" not found: spec.triggers.TriggerBinding
```